panther_description/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(panther_description)
+
+find_package(ament_cmake REQUIRED)
+
+install(DIRECTORY
+  config
+  meshes
+  urdf
```

```suggestion
  config
  meshes
  rviz
  urdf
```
might be useful when trying to find config by referencing the package